app/controllers/api/experimental/queries_controller.rb

```diff
@@ -155,11 +150,24 @@ def authorize_update_on_query
         actions.delete(:update) if changed.empty?
       end
 
-      allowed = actions.map(&:to_sym)
+      allowed = actions
+                .map(&:to_sym)
```

Comment:
Why mapping twice when we can just `...allowed?(original_query, action.to_sym)`?